be/src/formats/parquet/group_reader.h

```diff
@@ -49,23 +49,32 @@ struct GroupReaderParam {
     std::string timezone;
 
     vectorized::HdfsScanStats* stats = nullptr;
+
+    SharedBufferedInputStream* shared_buffered_stream = nullptr;
+
+    int chunk_size;
+
+    RandomAccessFile* file;
+
+    FileMetaData* file_metadata;
```

Comment:
better to give default value

Reply:
OK. but those values are assigned before used

```
Status FileReader::_init_group_readers() {
    const vectorized::HdfsScannerContext& fd_scanner_ctx = *_scanner_ctx;
    GroupReaderParam& param = _group_reader_param;
    param.tuple_desc = fd_scanner_ctx.tuple_desc;
    param.conjunct_ctxs_by_slot = fd_scanner_ctx.conjunct_ctxs_by_slot;
    param.read_cols = _read_cols;
    param.timezone = fd_scanner_ctx.timezone;
    param.stats = fd_scanner_ctx.stats;
    param.shared_buffered_stream = nullptr;
    param.chunk_size = _chunk_size;
    param.file = _file;
    param.file_metadata = _file_metadata.get();
```